bokehjs/src/lib/models/widgets/tables/data_cube.ts

```diff
@@ -115,6 +113,27 @@ export class DataCubeProvider extends TableDataProvider {
     this.groupingInfos = groupingInfos
     this.toggledGroupsByLevel = groupingInfos.map(() => ({}))
 
+    const row_indices: (number[] | number)[] = this.target.get_array("row_indices")
+    const labels: string[] = this.target.get_array("labels")
+
+    const parents: number[][] = []
+    const parent_labels: string[] = []
+    row_indices.forEach((indices: number[] | number, i: number) => {
```

Comment:
Explicit types are redundant:
```suggestion
    row_indices.forEach((indices, i) => {
```